test/models/bot/alegre_v2_test.rb

```diff
@@ -283,6 +283,13 @@ def teardown
     assert_equal JSON.parse(Bot::Alegre.get_sync(pm1).to_json), JSON.parse(response.to_json)
   end
 
+  test "should safe_get_sync" do
+    pm1 = create_project_media team: @team, media: create_uploaded_audio
+    response = {}
+    WebMock.stub_request(:post, "#{CheckConfig.get('alegre_host')}/similarity/sync/audio").with(body: {:doc_id=>Bot::Alegre.item_doc_id(pm1), :context=>{:team_id=>pm1.team_id, :project_media_id=>pm1.id, :has_custom_id=>true}, :url=>Bot::Alegre.media_file_url(pm1)}).to_return(body: response.to_json)
```

Comment:
Suggestion: I would simplify this because for this test the payload itself doesn't matter... something like:
```
WebMock.stub_request(:post, "#{CheckConfig.get('alegre_host')}/similarity/sync/audio").to_return(body: '{}')
```